sample.csv

```diff
@@ -1,9 +1,9 @@
-Organization,Repository,branch,Author's GitHub ID,Author's Display Name,Author's Git Author Name
-CS2103AUG2017-W09-B1,main,master,nbriannl,Nbr,
-CS2103AUG2017-W09-B1,main,master,zacharytang,Zac,Zachary Tang
-CS2103AUG2017-W09-B1,main,master,April0616,Fan,LAPTOP-7KFM2KSP\User;Fan Yuting
-CS2103AUG2017-W09-B1,main,master,CindyTsai1,Cin,YuHsuan
-CS2103JAN2018-F14-B1,main,master,lithiumlkid,Ahm,
-CS2103JAN2018-F14-B1,main,master,codeeong,Cod,
-CS2103JAN2018-F14-B1,main,master,jordancjq,Jor,
-CS2103JAN2018-F14-B1,main,master,lohtianwei,Loh,
+Repository's Location,Branch,Author's GitHub ID,Author's Display Name,Author's Git Author Name
+https://github.com/CS2103AUG2017-W09-B1/main.git,master,nbriannl,Nbr,
+https://github.com/CS2103AUG2017-W09-B1/main.git,master,zacharytang,Zac,Zachary Tang
+https://github.com/CS2103AUG2017-W09-B1/main.git,master,April0616,Fan,LAPTOP-7KFM2KSP\User;Fan Yuting
+https://github.com/CS2103AUG2017-W09-B1/main.git,master,CindyTsai1,Cin,YuHsuan
+https://github.com/CS2103JAN2018-F14-B1/main.git,master,lithiumlkid,Ahm,
+https://github.com/CS2103JAN2018-F14-B1/main.git,master,codeeong,Cod,
+https://github.com/CS2103JAN2018-F14-B1/main.git,master,jordancjq,Jor,
+https://github.com/CS2103JAN2018-F14-B1/main.git,master,lohtianwei,Loh,
```

Comment:
btw, since we're changing the format, I would like to take this opportunity to replace them with our organization as well.
`CS2103AUG2017-W09-B1` -> `testrepo-Beta`
`CS2103JAN2018-F14-B1` -> `testrepo-Delta`